walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,16 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+public void testGreetingTrimsWhitespace() {
+  Greeter greeter = new Greeter();
+
+  String greeting = greeter.greet("   Ada   ");
+
+  // Whitespace should be trimmed
+  Assert.assertEquals("Hello Ada", greeting);
```

Comment:
Reverse the order - left side is always the output generated while the right side is the expected output.